app/fridge/page.tsx

```
@@ -93,6 +97,10 @@ export default function FridgePage() {
     load();
   }, []);
 
+  useEffect(() => {
+    if (isEditMode && items.length === 0) exitEditMode();
```

ESLint

[error] Error: Cannot access variable before it is declared

`exitEditMode` is accessed before it is declared, which prevents the earlier access from updating when this value changes over time.

/home/jailuser/git/app/fridge/page.tsx:101:43
   99 |
  100 |   useEffect(() => {
> 101 |     if (isEditMode && items.length === 0) exitEditMode();
      |                                           ^^^^^^^^^^^^ `exitEditMode` accessed before it is declared
  102 |   }, [items, isEditMode]);
  103 |
  104 |   const filteredItems = useMemo(() => {

/home/jailuser/git/app/fridge/page.tsx:238:3
  236 |   }
  237 |
> 238 |   function exitEditMode() {
      |   ^^^^^^^^^^^^^^^^^^^^^^^^^
> 239 |     setSelectedIds([]);
      | ^^^^^^^^^^^^^^^^^^^^^^^
> 240 |     setIsEditMode(false);
      | ^^^^^^^^^^^^^^^^^^^^^^^
> 241 |   }
      | ^^^^ `exitEditMode` is declared here
  242 |
  243 |   async function handleBulkDelete() {
  244 |     if (selectedIds.length === 0) return;

(react-hooks/immutability)